### PR TITLE
Updated workorders to read volume from deeds

### PR DESCRIPTION
**Pull request by trall541**
workorders will pull the volume from a deed where appropriate instead of using analyze, this removes the minimum appraisal required to use your own ingots and gives a slight optimization in removing analyze RT

### CODE REVIEW

File: workorders.lic [hunk 1]
```diff
@@ -623,11 +630,11 @@ class WorkOrders
         stow_hands
         exit
       else
+		volume = deed_ingot_volume
         fput('tap deed')
         pause
         fput("get #{@use_own_ingot_type} ingot") unless checkleft || checkright
       end
-      volume = ingot_volume
```
Comment (rpherbig) [inline workorders.lic]: Do you need to do another `unless volume` check here?

File: workorders.lic [hunk 2]
```diff
@@ -606,13 +611,15 @@ class WorkOrders
...
       else
+        volume = deed_ingot_volume
         fput('tap deed')
         pause
         fput("get #{@use_own_ingot_type} ingot") unless checkleft || checkright
       end
     end
-
-    volume = ingot_volume
+    if(!volume)
```
Comment (rpherbig) [inline workorders.lic]: What do you think about `volume = ingot_volume unless volume`?

File: workorders.lic (hunk 1 above)
Comment (trall541) [reply workorders.lic]: this spot will only get here if the item is a deed, so its really not needed at all

File: workorders.lic (hunk 2 above)
Comment (trall541) [reply workorders.lic]: that works too :)

**Pull request merged**